### PR TITLE
Add Dead or Alive: Dimensions

### PR DESCRIPTION
**Pull request by FicheeSS**
Adds Dead or Alive: Dimensions.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/dead-or-alive-dimensions/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Dead or Alive: Dimensions"
+description = "Dead or Alive: Dimensions (DOAD) is a fighting game developed by Team Ninja and released by Tecmo Koei and in some regions by Nintendo for the Nintendo 3DS . Originally planned as a launch title for the 3DS but delayed due to 2011 Tōhoku earthquake and tsunami, DOAD makes use of the console's unique features. The game was well received by critics."
```
Comment (Leo626) [inline games/dead-or-alive-dimensions/game.dat]: The first sentence has an additional space before the period. The other sentences aren't needed. Here's how the description could be written.

> Dead or Alive: Dimensions (DOAD) is a fighting game developed by Team Ninja and released by Tecmo Koei and in some regions by Nintendo for the Nintendo 3DS. This made it the first Dead or Alive title for a Nintendo system, and the series's first fighting game for any handheld.

File: games/dead-or-alive-dimensions/game.dat [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000032000"
+region = "all"
```
Comment (Leo626) [inline games/dead-or-alive-dimensions/game.dat]: This game does not use `all` for the region.

File: games/dead-or-alive-dimensions/game.dat [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+release_date = "2011-05-24"
```
Comment (Leo626) [inline games/dead-or-alive-dimensions/game.dat]: This date is for `usa`.

File: games/dead-or-alive-dimensions/game.dat [hunk 4]
```diff
@@ -0,0 +1,9 @@
+title = "Dead or Alive: Dimensions"
+description = "Dead or Alive: Dimensions (DOAD) is a fighting game developed by Team Ninja and released by Tecmo Koei and in some regions by Nintendo for the Nintendo 3DS . Originally planned as a launch title for the 3DS but delayed due to 2011 Tōhoku earthquake and tsunami, DOAD makes use of the console's unique features. The game was well received by critics."
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000032000"
```
Comment (Leo626) [inline games/dead-or-alive-dimensions/game.dat]: This title ID is for `jpn`.

**Comment by CodingKoopa** [comment]
Please address the review comments, or this will be closed.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**